### ISSUE TITLE
make MultiLangStringView a custom element

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#369]
The MultiLangStringView would work really well as a custom element if it used `<slot>`s. It would be used this way:

```html
<dlx-mls>
  <p slot=eng>English text</p>
  <p slot=spa>texto en español</p>
</dlx-mls>
```

**Comment by vadekh** [comment]
@dwhieb so this would be instead of the <fieldset> system currently used in MultiLangString.js?

**Comment by dwhieb** [comment]
Yes and no. The MLSView would still use `<fieldset>` internally, but the user would provide content to it like I showed above. It would be analogous to how you put `<option>` elements inside a `<select>` tag, and then the browsers do the work in the shadow DOM to make all that functionality happen.

This might be a _little_ tricky to implement though, just because the MLSView internally would have to iterate through the `<p>` tags and create `<slot>` elements for each.

I'm honestly not 100% sure doing all this really gains us anything though. Might just close this issue if not. I could go either way. Would appreciate any thoughts!
